### PR TITLE
[Improvement][SQL] Query return number In SQL should be configurable

### LINKED ISSUE #5580: [Improvement][SQL] Query return number should be configurable
**Describe the question**
The SQL query result can only return 10,000 records, which are currently hard-coded

**Which version of DolphinScheduler:**
 -[1.3.6-Release]

**Describe alternatives you've considered**
I think it should be configurable.

### PR DESCRIPTION
**Pull request by echohlne**
## Purpose of the pull request
Try to make the `Number of records to return`  configurable, issue: closes #5580 
## Brief change log
+  front-end: A new text input box and the related processing has been added to the page to represent the number of records
+  changed the hard code in `SqlTask.java` by add a variable `limit` in `SqlParameters.java`

## Verify this pull request
Munual test works well，the test results have been saved in the document below:
[munual_test.docx](https://github.com/apache/dolphinscheduler/files/6644368/munual_test.docx)

### CODE REVIEW

**Comment by CalvinKirs** [comment]
Back-end code looks great.

**Comment by echohlne** [comment]
@CalvinKirs hi, I have added the corresponding unit test with the modifed code, but code analysis always give an error：
`Failed condition 27.3% 27.3% Coverage on New Code (is less than 33%)`, what should I do?

**Comment by CalvinKirs** [comment]
> @CalvinKirs hi, I have added the corresponding unit test with the modifed code, but code analysis always give an error：
> `Failed condition 27.3% 27.3% Coverage on New Code (is less than 33%)`, what should I do?

Don't worry, I will let other community partners review the UI part.

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/sql.vue [hunk 1]
```diff
@@ -43,6 +43,13 @@
         </div>
       </div>
     </m-list-box>
+    <m-list-box>
+        <div slot="text"><strong class='requiredIcon'>*</strong>{{$t('Max Numbers Return')}}</div>
+        <div slot="content">
+          <el-input type="input" :disabled="isDetails" size="medium" v-model="limit" :placeholder="$t('Max Numbers Return placeholder')">
+          </el-input>
+        </div>
+    </m-list-box>
```
Comment (zhuangchong) [inline dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/sql.vue]: This limit number of rows only needs to be displayed in the query scenario, and it is recommended that the backend also set the initial value of limit at the same time

**Review by zhuangchong — CHANGES_REQUESTED** [review]

File: dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/sql.vue (hunk 1 above)
Comment (echohlne) [reply dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/tasks/sql.vue]: @zhuangchong Thanks for youre review.
Sorry I forget to merge some code. I have updated by add a condition `<m-list-box v-show="sqlType === '0'">`

**Review by zhuangchong — APPROVED** [review]
LGTM.

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**